photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimatorEvaluationFunction.scala

```diff
@@ -14,35 +14,44 @@
  */
 package com.linkedin.photon.ml.estimators
 
-import scala.collection.SortedMap
-import scala.collection.mutable.Queue
+import scala.collection.mutable
 
 import breeze.linalg.DenseVector
 import org.apache.spark.sql.DataFrame
 
-import com.linkedin.photon.ml.estimators.GameEstimator.GameResult
+import com.linkedin.photon.ml.estimators.GameEstimator.{GameOptimizationConfiguration, GameResult}
 import com.linkedin.photon.ml.hyperparameter.EvaluationFunction
-import com.linkedin.photon.ml.normalization.NormalizationContext
-import com.linkedin.photon.ml.optimization.game.GameModelOptimizationConfiguration
+import com.linkedin.photon.ml.optimization.game.{FactoredRandomEffectOptimizationConfiguration, FixedEffectOptimizationConfiguration, RandomEffectOptimizationConfiguration}
 
 /**
- * Evaluation function implementation for GAME
+ * Evaluation function implementation for GAME.
  *
  * An evaluation function is the integration point between the hyperparameter tuning module and an estimator, or any
  * system that can unpack a vector of values and produce a real evaluation.
  */
 class GameEstimatorEvaluationFunction(
```

Comment:
@joshvfleming 

You may be interested in the new method for constructing hyperparameter vectors from configurations (given that the configurations have changed so much).